hack/make/validate-test

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Make sure we're not using gos' Testing package any more in integration-cli
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^vendor/' || true) )
```

Comment:
yes if we want to check all files instead of just the ones changed by the current PR.
I was following the pattern set by the other validate-* scripts.  If you're ok with the inconsistency I'll change it.
Just let me know.